cmd/kpod/spec.go

```diff
@@ -17,6 +18,30 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func addPidNS(config *createConfig, g *generate.Generator) error {
+	pidMode := config.pidMode
+	if pidMode.IsHost() {
+		return g.RemoveLinuxNamespace("pid")
+	}
+	if pidMode.IsContainer() {
+		ctr, err := config.runtime.LookupContainer(pidMode.Container())
+		if err != nil {
+			return errors.Wrapf(err, "container %q not found", pidMode.Container())
+		}
+		if _, err := ctr.State(); err != nil {
+			return err
+		}
+		return libpod.ErrNotImplemented
+		/*
+			pidNsPath := fmt.Sprintf("/proc/%d/ns/pid", state.Pid)
```

Comment:
Is this commented code needed or will it be added shortly?  A TODO comment maybe if it's going to stay?

Reply:
Should add a TODO.